plugins/module_utils/core.py

```diff
@@ -38,11 +38,15 @@
 from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry
 from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
 
-# Replicated common.py filefrom amazon.aws because the client agent string will be
-# released in amazon.aws 6.0.0
-from ansible_collections.amazon.cloud.plugins.module_utils.common import (
+from ansible_collections.amazon.aws.plugins.module_utils.common import (
     set_collection_info,
 )
+from ansible_collections.amazon.cloud.plugins.module_utils.common import (
+    AMAZON_CLOUD_COLLECTION_NAME,
+)
+from ansible_collections.amazon.cloud.plugins.module_utils.common import (
```

Comment:
Can not we entirely remove the common.py file?

Reply:
no because we are keeping the same module architecture of others collections (amazon.aws and community.aws)